index.html

```diff
@@ -9,10 +9,10 @@
         <div>
             <a href="/">
                 <!--/ this is the root = home-->
-                <img src="Cat.jpg"
-                     alt="Cat">
+                <img src="image/Cat.jpg"
+                     alt="logo Cat">
             </a>
-            <h3>Lorem ipsum dolor sit amet</h3>
+            <h2>Lorem ipsum dolor sit amet</h2>
             <p>other</p>
         </div>
         <nav>
```

Comment:
In nav tag i add list (ul>li>a)

Reply:
done

Reply:
isn't great correction. modify your menu like this:
<nav>
<ul>
<li><a href="/">home</a></li>
<li>other</li>
</ul>
</nav>

Reply:
done with the mentionned corrections
thanks